tests/basic/upgrade_test.go

```diff
@@ -157,6 +157,20 @@ var _ = Describe("{UpgradeVolumeDriver}", func() {
 				close(stopSignal)
 			}()
 
+			var vQourumError error
+			opver, err := optest.GetPxOperatorVersion()
+			if err == nil && opver.GreaterThanOrEqual(ParallelUpgradeVersion) {
+				log.Info("Starting volume quorum validation for Portworx upgrade .......")
+
+				stopVolumeQuorumValidationSignal := make(chan struct{})
+				go DoVolumeQuorumValidation(stopVolumeQuorumValidationSignal, &vQourumError)
+				defer func() {
+					close(stopVolumeQuorumValidationSignal)
```

Comment:
We can always validate, but the number of nodes that will be upgraded will not be always same what customer has configured. Since this completely depends on volume placement strategy, like where volume replicas are residing, it is difficult to validate that always specific number of nodes are being upgraded or not.